server/app/views/questiontypes/QuestionBaseFragment.html

```diff
@@ -0,0 +1,3 @@
+<th:block th:fragment="validationErrors (validationErrors, messages)">
+    <div th:each="error: ${validationErrors}" th:text="${error.getMessage(messages)}"></div>
```

Comment:
does something like this work?
```
<div th:each="error: ${validationErrors}" th:text="#{${error.key}(${error.args})}"></div>
```
From the last example under https://www.thymeleaf.org/doc/tutorials/3.1/usingthymeleaf.html#messages

Reply:
good idea I got a variant of this working